### PR TITLE
feat(index): add recent news

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

Add recent news to index.

image

### CODE REVIEW

File: themes/navy/source/css/_partial/index.styl [hunk 1]
```diff
@@ -45,6 +45,39 @@
...
+  display: flex
+  flex-flow: column
+  @media mq-normal
+    flex-flow: row wrap
+
+.intro-news-wrap
+  padding: 36px 30px
+  text-decoration: none
+  &:hover
+    background-color: #25a1f0
```
Comment (curbengh) [inline themes/navy/source/css/_partial/index.styl]: recommend to use existing variable "color-link-hover".

File: themes/navy/source/css/_partial/index.styl [hunk 2]
```diff
@@ -45,6 +45,39 @@
   @media mq-normal
     display: block
 
+#intro-news-list
+  background-color: #10171e
```
Comment (curbengh) [inline themes/navy/source/css/_partial/index.styl]: "black" is sufficient.

File: themes/navy/source/css/_partial/index.styl [hunk 3]
```diff
@@ -45,6 +45,39 @@
...
+  &:hover
+    background-color: #25a1f0
+  @media mq-normal
+    flex: 0 0 50%
+  @media screen and (min-width: 960px)
+    flex: 0 0 25%
+
+.intro-news-time
+  font-size: 14px
+  color: #888
```
Comment (curbengh) [inline themes/navy/source/css/_partial/index.styl]: I would have preferred banner font color "rgba(255,255,255,.8)", but it maybe too bright.

File: themes/navy/source/css/_partial/index.styl [hunk 4]
```diff
@@ -45,6 +45,39 @@
...
+
+.intro-news-wrap:hover .intro-news-time
+  color: #ddd
```
Comment (curbengh) [inline themes/navy/source/css/_partial/index.styl]: `"color-link-hover"`

File: themes/navy/source/css/_partial/index.styl (hunk 2 above)
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: I have tried black but it seems not fit navy theme design language.

File: themes/navy/source/css/_partial/index.styl [hunk 5]
```diff
@@ -45,6 +45,39 @@
...
+  margin: -1px 0
+
+#intro-news-flex
+  display: flex
+  flex-flow: column
+  @media mq-normal
+    flex-flow: row wrap
+
+.intro-news-wrap
+  padding: 36px 30px
```
Comment (YoshinoriN) [inline themes/navy/source/css/_partial/index.styl]: I think `36px` is a little too high for a smartphone. How do you think?

navy

File: themes/navy/source/css/_partial/index.styl (hunk 1 above)
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Updated.

File: themes/navy/source/css/_partial/index.styl (hunk 5 above)
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Now it is `24px` for smartphone, and remains `36px` on tablet & desktop.

File: themes/navy/source/css/_partial/index.styl (hunk 3 above)
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Changed to `rgba(255,255,255,.6)`

File: themes/navy/source/css/_partial/index.styl (hunk 4 above)
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Changed to `rgba(255,255,255,.8)`

File: themes/navy/source/css/_partial/index.styl (hunk 2 above)
Comment (curbengh) [reply themes/navy/source/css/_partial/index.styl]: My poor monitor can't differentiate black and #10171e

Anyway, why not use `lighten(color-navy, 10%)` as used in #banner-start-command.

Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: The `#10171e` I choose is more like `darken(color-navy, 10%)`. Maybe `lighten(color-navy, 10%)` is better.

Comment (curbengh) [reply themes/navy/source/css/_partial/index.styl]: I'm a bit reluctant to introduce new color. If darker background is preferred, then it's better to use `darken(color-navy, 10%)`.

Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Updated.

**Pull request merged**